documentation/source/Usecases/mongodb.rst

```diff
@@ -0,0 +1,431 @@
+::
+
+    sudo cat /etc/iscsi/initiatorname.iscsi
+    sudo service open-iscsi status
+
+* Install the following dependent packages to run mongodb-integrated sysbench input/output tool on any one of the Kubernetes nodes.
+  ::
+
+    sudo apt-get install <packagename>:
+
+    make
+    libsas12-dev
+    libssl-dev
+    libmongoc-dev
+    libbson-dev
+
+1. Run OpenEBS Operator
+-------------------------
+Download the latest OpenEBS operator files and sample MongoDB statefulset specification yaml on the Kubernetes master from the OpenEBS git repository.
+::
+    
+    git clone https://github.com/openebs/openebs.git
+    cd openebs/k8s
+
+Apply the openebs-operator.yml on the Kubernetes cluster. This creates the maya api-server and OpenEBS provisioner deployments.
+::
+    kubectl apply -f openebs-operator.yaml
+
+Add the OpenEBS storage classes, that can be used by developers to map a suitable storage profile for their applications in their respective persistent volume claims.
+::
+
+   kubectl apply -f openebs-storageclasses.yaml
+
+Check whether the deployments are running successfully.
+::
+
+    test@Master:~$ kubectl get deployments
+    NAME                                            DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+    maya-apiserver                                  1         1         1            1           5s
+    openebs-provisioner                             1         1         1            1           6s
+
+    testk@Master:~$ kubectl get pods
+    NAME                                   READY     STATUS    RESTARTS   AGE
+    maya-apiserver-1089964587-x5q15        1/1       Running   0          10s
+    openebs-provisioner-1149663462-5pdcq   1/1       Running   0          9s
+
+Check whether the storage classes are applied successfully.
+::
+
+    test@Master:~$ kubectl get sc
+    NAME                 TYPE
+    openebs-cassandra    openebs.io/provisioner-iscsi
+    openebs-es-data-sc   openebs.io/provisioner-iscsi
+    openebs-jupyter      openebs.io/provisioner-iscsi
+    openebs-kafka        openebs.io/provisioner-iscsi
+    openebs-mongodb      openebs.io/provisioner-iscsi
+    openebs-percona      openebs.io/provisioner-iscsi
+    openebs-redis        openebs.io/provisioner-iscsi
+    openebs-standalone   openebs.io/provisioner-iscsi
+    openebs-standard     openebs.io/provisioner-iscsi
+    openebs-zk           openebs.io/provisioner-iscsi
+
+2. Deploy the mongo-statefulset with OpenEBS storage
+------------------------------------------------------
+
+Use OpenEBS as persistent storage for the MongoDB statefulset by selecting an OpenEBS storage class in the persistent volume claim. A sample MongoDB statefulset yaml (with container attributes and pvc details) is available in the OpenEBS git repository.
+
+The number of replicas in the statefulset can be modified as required. The following example uses two replicas. The replica count can be edited in the statefulset specification.
+::
```

Comment:
The number of replicas in the statefulset can be modified as required. The following example uses three replicas

Reply:
Done! Thought as much :)